packages/server/src/api/rest/index.ts

```diff
@@ -719,6 +719,18 @@ class RequestHandler extends APIHandlerBase {
         const parsed = this.createUpdatePayloadSchema.parse(body);
         const attributes: any = parsed.data.attributes;
 
+        // Map in the compound id relationships as attributes, as they are expected by the zod schema
```

Comment:
 Hi @thomassnielsen , I think we should probably validate using a zod schema that doesn't include fk fields here. The purpose should be validating rules declared on the data fields.

Shall I make some tweaks directly?

Reply:
That makes sense. Yes, go ahead and make the changes. I'll read them to learn afterwards 😄